src/autolabel/transforms/pdf.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Dict, Any
+
+from pdf2image import convert_from_path
```

Comment:
do these packages need to be installed separately as well?

Reply:
Yep, just moved them